docs/source/using/addons.adoc

```diff
@@ -181,48 +176,46 @@ By default, an add-on has the priority 0. Add-ons with a lower priority value ar
 [[example-priority-list]]
 .Example: List command output with explicit priorities
 
+In this example, the *anyuid*, *registry*, and *eap* add-ons are enabled with the respective priorities of 0, 5 and 10.
```

Comment:
Same as with other tables and examples. This text belongs outside of the example/table scope. I double checked with the rest of docs.openshift.org and there it is also outside.